vshard/storage/init.lua

```diff
@@ -1251,10 +1257,12 @@ local function recovery_service_f(service)
         end
 
     ::sleep::
+        if not has_error then
```

Comment:
Thanks for noticing, indeed looks bad. I've changed it so now when something is not recovered, we set a status `recovering`. I didn't want to use `error` status because it is not really an error, when some buckets are just being transferred normally.

Reply:
Yes, it's better